### ISSUE TITLE
Q: StartNoloServer is optional? (driver_huaweinolo)

### ISSUE THREAD

**Issue by m-7761** [NOLOVR/NoloDeviceSDK#2]
The sample project (OpenVR) doesn't use StartNoloServer:
```
void NoloDeviceManager::InitNoloDevice()
{
	//StartNoloServer();
	SetEventListener(this);
	OpenNoloZeroMQ();
}
```
What happens in this case? Is it equivalent to doing `StartNoloServer();` before using any other API? Thank you.

Is it possible future (or current) SDK will find another way to access the Nolo device? I.e. Is it recommended to not use this API?

**Comment by sols000** [comment]
Sometimes, We can receive data from NoloHome, In that case, we should not call StartNoloServer().

**Comment by m-7761** [comment]
Hmm, this is very interesting! Is Nolo Assistant replace by Nolo Home? 

I would prefer my software's users to use Nolo Home if possible. 

P.S. I think my Nolo Assistant was using a lot of the CPU and making the host machine to spin its fans at maximum. I don't think it normally does this. But it was maybe because the Nolo devices were plugged in and it was showing their battery status. When I closed it this stopped. (I should install Nolo Home. I tried to use SteamVR but it was not good for my system. Maybe Nolo Home will work with desktop. My VR software works very well. I do not want to develop software for expensive game computers only.)

**Comment by sols000** [comment]
Yes, Nolo Assistant is about to replace by Nolo Home. But not complete.
